cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -308,7 +291,13 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
                 is_tumour=rna_sample.is_tumour,
             )
 
-            self._check_cases_single_connected(dna_cases, rna_sample)
+            upload_dna_cases.update(dna_cases)
```

Comment:
What is `upload_dna_cases` used for? Why can't we not just use `dna_cases`?

Reply:
I agree here, but as we are rewriting the code now from commit "allow upload multiple splice junction files" to allow uploading from RNA cases with multiple samples to DNA cases with multiple samples, we will need the upload_dna_cases list.